exonum/src/events/noise/wrappers/sodium_wrapper/wrapper.rs

```diff
@@ -160,8 +162,12 @@ impl NoiseWrapper {
     // of AEAD authentication data. Therefore to calculate an actual message
     // length we need to subtract `TAG_LENGTH` multiplied by messages count
     // from `data.len()`.
+    //
+    // f32 precision is enough to calculate message length.
```

Comment:
Cast to `f32` is not needed. `ceil` can be easily implemented for integer arguments.